README.md

```diff
@@ -91,6 +91,82 @@ class MyComponent extends Component {
 }
 ```
 
+# Code organization
+
+## Modules
+
+Actions, reducers, and potential helpers supporting the same functionality should be regrouped in a module folder. 
+
+> Why ? Action and reducers are by their very nature tightly coupled. When separated, refactoring and adding new features leads to editing of several files with tiny changes. With modules, this is simplified since changes are made in one file instead of several.
+
+If you develop functionalities related to greetings, here is how you can structure your folders :
+
+```
+src
+├── greetings
+│   └── components
+│        └── __tests__
+│        └── Greeting.jsx
+│    └── redux
+│        └── index.js
+```
+
+<details>
+    <summary>See more</summary>
+<p>
+
+##### Dumb component 
+
+`src/greetings/components/Greeting.jsx`
+```js
+export default ({ name }) => <div>Hello { name } !</div>
+```
+
+##### Redux related
+
+`src/greetings/redux/index.js`
+
+```js
+import Greeting from 'components/Greeting'
```

Comment:
Without anticipating the build system configuration of JavaScript modules, we must use relative paths for all local packages.

```js
import Greeting from '../components/Greeting'
```